README.md

```diff
@@ -24,3 +24,62 @@ All other requests require a valid JWT to be sent in the `Authorization` header
 GET https://localhost:5001/Addresses?postcode=1
 Authorization: Bearer <JWT TOKEN>
 ```
+## Running the API locally with other Services
+In order to run the API locally, the following steps may need to be taken to exclude some functionality.
+
+Only make the changes necessary to enable you to work on functionality locally and bear in mind the adjustments you make locally when diagnosing issues.
+Take care to not commit any of these local-only changes.
+
+In your local .env file, if you wish to connect to the Housing Management API, set
+AUTHENTICATION_IDENTIFIER to the same value as the front end and the Housing Management API.
+Set ADDRESSES_API_URL to the location of your locally running Housing Management API:
+```
+export ADDRESSES_API_URL=https://localhost:6001/
+```
+
```

Comment:
It would be good to mention the equivalent and respective changes could be made to connect to the Scheduling API too (i.e. set `SCHEDULING_API_URL` instead)